### DISCUSSION TITLE
Dockerless wg-easy / systemd-Service

### DISCUSSION THREAD

**Discussion by kaaax0815** [wg-easy/wg-easy#1360] [Polls]
With the new UI, using the Service will be harder.
Previously the code was in "raw" node javascript.
But now its written Typescript and Nuxt and requires a build step.
This would require users to install build tools like pnpm which is not something most users want to do.
We could also distribute prebuilt files with GH Actions.

Docker Users (or Podman, Portainer, etc) are unaffected by this

**Comment by pheiduck** [comment]
I would decide to retire the systemd service do it's add more work to the project it can be still used as a specific use case, but it will not be official supported.
